libs/net/services/Helpers/ApiService.cs

```diff
@@ -448,11 +448,16 @@ public async Task<IEnumerable<TNO.API.Areas.Services.Models.Ingest.IngestModel>>
         try
         {
             var url = Options.ApiUrl.Append($"services/contents/{id}/image");
-            return await RetryRequestAsync(async () => await Client.GetAsync<string>(url));
+            return await RetryRequestAsync(async () => await Client.GetAsync<string>(url,
```

Comment:
Think this through.
You are asking to `RetryRequestAsync` which means it will attempt to retry a failure `X` times.  
Then you're overriding the error handling of a request to log a warning on a `BadRequest` response, and then return a `true` which will result in a `null` being returned by the original `Client.GetAscyn()`.
Then you have a `try+catch` around all of this.

a) Why have the request ability if you're not going to use it?
b) Don't have an unnecessary `try+catch` if it doesn't make any sense.